sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Updated the `Microsoft.Azure.Amqp` dependency to 2.6.4, which includes several fixes.  One notable fix addresses an obscure race condition when a cancellation token is signaled while service operations are being invoked concurrently which caused those operations to hang.
```

Comment:
nit - was the version updated from 2.6.4 to 2.6.5 or am I reading that wrong?
```suggestion
- Updated the `Microsoft.Azure.Amqp` dependency to 2.6.5, which includes several fixes.  One notable fix addresses an obscure race condition when a cancellation token is signaled while service operations are being invoked concurrently which caused those operations to hang.
```

Reply:
You commented just before I pushed the fix for that.   😄